.github/CODEOWNERS

```diff
@@ -13,6 +13,10 @@ people/**/*.toml
 repos/**/*.toml
 teams/**/*.toml
 
+# Do not require admin approvals for Markdown file modifications.
+*.md
+/**/*.md
```

Comment:
```suggestion
```
based on [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) example, `/**/*.md` shouldn't required.

Reply:
Good point, removed it.